ibm/mas_devops/roles/ocp_provision/tasks/providers/roks.yml

```diff
@@ -112,11 +112,21 @@
 
 # 8. Add a GPU worker node if ocp_provision_gpu is true
 # -----------------------------------------------------------------------------
+- name: "roks : Debug Information for GPU worker"
+  when:
+    - ocp_provision_gpu is defined
+    - ocp_provision_gpu|bool == true
+  debug:
+    msg:
+      - "GPU woker pool name ................. {{ gpu_wpool_name }}"
+      - "GPU workers .................. {{ gpu_workers }}"
```

Comment:
Can you add extra dots to this line so it lines up with the rest, see https://github.com/ibm-mas/ansible-devops/blob/master/CONTRIBUTING.md#align-debug-messages

Reply:
Thanks for sharing the document. very useful :)